### ISSUE TITLE
Make the biome blending configurable

### ISSUE THREAD

**Issue by Kichura** [FionaTheMortal/Better-Biome-Blend#56]
By default, this mod does not allow to modify the biome blending as it is stuck at 29x29 maxmium in vanilla's view; however when using options.txt to put the value to 14, it can go up to 29x29 anyways it appears to be "unused".

So my question is, is it possible to have this mod be configurable so that the blending isn't hard-coded and instead have the user decide how much blending they wish to use instead? (can be config or use vanilla api for that)

**Comment by FionaTheMortal** [comment]
Yes version 1.18 is hard coded to 29x29. The reason is that biomes are now 3D which made the problem harder to solve.

Reducing the radius wouldn't really make a difference for performance though. What other reasons are there to change it? Is it a visual or practical motivation?

**Comment by Julienraptor01** [comment]
Personally it's a visual motivation as i think it's too blended with 29x29.
And i would like to be able to disable biome blending completely because sometime you want to know what is the limit of the biome without having to use f3
